[Concept Entry] C keywords (#6027)
Codecademy/docs: content/c/concepts/keywords/keywords.md

---
Title: 'Keywords'

> Description: 'Reserved words that have predefined meanings and purposes in the programming language and cannot be used as identifiers.'

Reviewer: ```suggestion
Description: 'Reserved words with predefined meanings and purposes in the C programming language and cannot be used as identifiers.'
```
